code/modules/mob/living/carbon/human/human_grabs.dm

```diff
@@ -9,7 +12,7 @@
 			to_chat(grabber, SPAN_WARNING("\The [src] is missing that body part!"))
 			return FALSE
 		if(grabber == src)
-			var/using_slot = get_active_held_item_slot()
+			var/using_slot = get_empty_hand_slot()
```

Comment:
This is going to introduce inconsistencies since it will no longer fail when your main hand is occupied.

Reply:
I guess failing to grab yourself with ctrl-click sometimes is okay?

Reply:
(or I could pass defer_hand to yet another proc)

Reply:
Would prefer you pass the arg honestly.